packages/vertica-nodejs/mochatest/integration/client/copy-local-stdin-tests.js

```diff
@@ -0,0 +1,113 @@
+'use strict'
+const vertica = require('../../../lib')
+const assert = require('assert')
+const path = require('path')
+const fs = require('fs')
+
+describe('Running Copy From Local Stdin Commands', function () {
+  // global pool to use for queries
+  const pool = new vertica.Pool()
+
+  // global file names and paths
+  const goodFileName = "copy-good.dat"
+  const badFileName = "copy-bad.dat"
+  const goodFilePath = path.join(process.cwd(), goodFileName);
+  const badFilePath = path.join(process.cwd(), badFileName)
```

Comment:
This reads as a file that does not exist.

Reply:
I have updated the names